BaikalDataGenerator/Source/render.cpp

```diff
@@ -342,17 +364,12 @@ void Render::SaveOutput(OutputDesc desc,
         {
             float3 val = output_data[(height - 1 - y) * width + x];
             val *= (1.f / val.w);
```

Comment:
Let's add the following comment into the code:
"The 4-th pixel component is a count of accumulated samples. It can be different for every pixel in case of adaptive sampling. So, we need to normalize pixel values here".